db/views/return_articles_v01.sql

```diff
@@ -0,0 +1,13 @@
+Select Articles.id as article_id, 
```

Comment:
Please use same typing - for tag words use uppercase
```
SELECT
INNER JOIN
FROM
```

Reply:
fixed